docs/in-depth.md

```diff
@@ -38,6 +38,12 @@ Note that this explicitly excludes the [Babylon](https://github.com/babel/babylo
 and only measures the throughput of the actual transformations. The parser is tested
```

Comment:
We’ll also need to update the example in `README.md`: https://github.com/v8/web-tooling-benchmark#running